code/modules/clothing/head/helmet.dm

```diff
@@ -178,7 +178,7 @@
 	toggle_message = "You pull the visor down on"
 	alt_toggle_message = "You push the visor up on"
 	armor_type = /datum/armor/toggleable_riot
-	flags_inv = HIDEEARS|HIDEFACE|HIDESNOUT
+	flags_inv = HIDEHAIR|HIDEEARS|HIDEFACE|HIDESNOUT  
```

Comment:
why is there excess whitespacing here? please trim it out

Reply:
fixed